src/molecules/radio/Radio.js

```diff
@@ -1,6 +1,10 @@
 // @vue/component
+import ALabel from '../../atoms/label/Label.vue'
```

Comment:
You can not add import in js file. As you can see in other components we add a child component to vue file to avoid inheritance  issue in the parent component.

Reply:
example here https://github.com/SnowdogApps/alpaca-ui/blob/develop/src/molecules/radio-wrapper/RadioWrapper.vue